assets/css/style.css

```diff
@@ -473,12 +476,163 @@ a {
     color: #344563;
   }
 
+  .home-page header {
+    background-image:
+      url("../images/desktop/Frame.svg"),
+      url("../images/desktop/Frame-1.svg"),
+      url("../images/desktop/Frame-2.svg"),
+      url("../images/desktop/Frame-3.svg"),
+      url("../images/desktop/Frame-4.svg"),
+      url("../images/desktop/Frame-5.svg"),
+      url("../images/desktop/Frame-6.svg"),
+      url("../images/desktop/desktop-back.svg");
+    background-position:
+      0% 26%,
+      77% 0%,
+      41% 62%,
+      82% 76%,
+      104% 19%,
+      18% 75%,
+      37% 7%,
+      57% 0;
+    background-repeat: no-repeat;
+    height: 1121.3px;
+  }
+
   #banner {
-    height: 916px;
-    margin-left: 18%;
-    margin-right: 18%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 10%;
+  }
+

```

Comment:
- [ ] Great work, you deserve a round of applause, you did an excellent job, however, I have some improvement to suggest you can add to the header, see this image below, I think if you add a white background to it, it will make your website more professional on some point.
<img width="329" alt="hder" src="https://user-images.githubusercontent.com/79058364/154718605-2fad0357-ad5f-424b-97f4-4762c8df2b9e.png">

-  as you can see, when the user scrolls, the header is kinda confused with the actual section text, please fix that